pkg/asset/manifests/content/bootkube/cvo-overrides.go

```diff
@@ -35,10 +35,6 @@ overrides:
   namespace: openshift-cluster-network-operator
   name: cluster-network-operator
   unmanaged: true
-- kind: Deployment                    # this conflicts with tectonic-ingress-controller-operator
-  namespace: openshift-cluster-ingress-operator
-  name: cluster-ingress-operator
-  unmanaged: true
 - kind: ServiceAccount                # missing run level 0 on the namespace and has 0000_08
```

Comment:
Did you mean to drop both the deployment and service account? As per comments in #415?

Reply:
no only the deployment - the service account is for the cluster-dns-operator.